Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang AS build
+# renovate: name=kubernetes-sigs/cluster-api-provider-openstack repo=https://github.com/kubernetes-sigs/cluster-api-provider-openstack.git branch=release-0.11
+ARG CAPO_GIT_REF=dcf91abef5c8500c1d8894a8e0f00d136977d27b
+ADD https://github.com/kubernetes-sigs/cluster-api-provider-openstack.git#${CAPO_GIT_REF} /src
+WORKDIR /src
+COPY /patches /patches
+RUN git apply /patches/*.patch
```

Comment:
I think instead of pinning to git repo, I would rather we point to the latest v0.11.x release and then add the patch on top of it.